global.R

```diff
@@ -35,9 +35,11 @@ shhh(library(arrow))
 shhh(library(dplyr))
 shhh(library(tidyr))
 shhh(library(forcats))
+shhh(library(validate))
```

Comment:
validate isn't a package, it's a function that comes in the shiny package so need to delete this line (the app gives an error if you try to run it with it in)